server/index.js

```diff
@@ -6,8 +7,15 @@ const app = express();
 
 app.use('/', express.static(path.join(__dirname, '../public')));
 
-app.get('/test', (req, res) => {
-  res.send('Hello World');
+app.get('/details/:propertyId', (req, res) => {
+  db.getDetails(req.params.propertyId)
+    .then((details) => {
+      console.log(details[0]);
+      res.send(details[0]);
```

Comment:
thanks for the feedback!

details is returned from the getDetails function. getDetails uses async/await so it behaves like a promise, whatever it returns is available for .then() chaining. i think it's an array because the mongoose Model.find() method returns an array of documents that match the search parameters, but i'm searching by propertyId. the array should only have document.